contents/docs/integrate/_snippets/install-flutter-manual.mdx

```diff
@@ -0,0 +1,106 @@
+### Manual installation
+
+First, add `posthog_flutter` to your `pubspec.yaml`:
+
+```yaml file=pubspec.yaml
+# rest of your code
+
+dependencies:
+  flutter:
+    sdk: flutter
+  posthog_flutter: ^4.0.1
+
+# rest of your code
+```
+
+Then complete the manual setup for each platform:
+
+#### Android setup
+
+Add your PostHog configuration to your `AndroidManifest.xml` file located in the `android/app/src/main`:
+
+```xml file=android/app/src/main/AndroidManifest.xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="your.package.name">
+    <application>
+        <!-- ... other configuration ... -->
+        <meta-data android:name="com.posthog.posthog.AUTO_INIT" android:value="false" />
+    </application>
+</manifest>
+```
+
+And setup the SDK manually:
+
+```dart
+import 'package:flutter/material.dart';
+
+import 'package:posthog_flutter/posthog_flutter.dart';
+
+Future<void> main() async {
+  // init WidgetsFlutterBinding if not yet
+  WidgetsFlutterBinding.ensureInitialized();
+  final config = PostHogConfig('YOUR_API_KEY_GOES_HERE');
```

Comment:
instead of `YOUR_API_KEY_GOES_HERE`, use the place holder that replaces this by a real api key, check docs